<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="randomPB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="seedL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="centerL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seedLE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32769</number>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="widthL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="octavesL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Octaves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="amplitudeL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Amplitude:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4097</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4097</number>
    </property>
   </widget>
   <widget class="QLabel" name="xCenterL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yCenterL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="octavesSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="editOtavesPB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="scaleSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="centerXSB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="centerYSB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="amplitudeSB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="outputL">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="imagePB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export png</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
